File: ETour-DotNet/Models/Notification.cs
namespace ETour.Api.Models;

public class Notification
{
    public int NotificationId { get; set; }
    public int? UserId { get; set; }
    public User User { get; set; }
    public string Title { get; set; }
    public string Message { get; set; }
    public bool? IsRead { get; set; }
    public DateTime? CreatedDate { get; set; }
}

File: E-Tour/src/main/java/com/etour/entities/Notification.java
package com.etour.entities;

import jakarta.persistence.*;
import lombok.*;

import java.time.LocalDateTime;

@Entity
@Table(name = "notification")
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
@EqualsAndHashCode(onlyExplicitlyIncluded = true)
public class Notification {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "notification_id")
    @EqualsAndHashCode.Include
    private Integer notificationId;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "user_id")
    @ToString.Exclude
    private User user;

    @Column(name = "title", length = 150)
    private String title;

    @Column(name = "message", length = 500)
    private String message;

    @Column(name = "is_read")
    private Boolean isRead;

    @Column(name = "created_date")
    private LocalDateTime createdDate;
}
